observe_configure_mac_script.sh

```diff
@@ -662,6 +662,9 @@ if [ "$telegrafinstall" == TRUE ]; then
 
   sudo cp "$sourcefilename" "$filename"
 
+  LC_ALL=C  sed -i '' 's/\[\[inputs\.kernel\]\]/#[[inputs.kernel]]/'  $filename
```

Comment:
fwiw, this sed command takes this input

```
[[inputs.kernel]] hello world
[[inputs.kernel]] hi friend
[[inputs.kernel]]
[[inputs.kernel]] hello world
[[inputs.kernel]] hello world
[[inputs.kernel]] hello world
fooooooo [[inputs.kernel]] hello world


this should stay the same

```

and changes it to this output

```
#[[inputs.kernel]] hello world
#[[inputs.kernel]] hi friend
#[[inputs.kernel]]
#[[inputs.kernel]] hello world
#[[inputs.kernel]] hello world
#[[inputs.kernel]] hello world
fooooooo #[[inputs.kernel]] hello world


this should stay the same

```

I think that's fine for this purpose.